docs/admin/kubeadm-config.md

```diff
@@ -0,0 +1,36 @@
+---
+approvers:
+- mikedanese
+- luxas
+- jbeda
+title: Kubeadm config
+notitle: true
+---
```

Comment:
For the generated topics under _kubeadm, I see the need for notitle:true. But in these manual topics, it should be fine to let the title come from the front matter. And then you wouldn't need to put the title inside {% capture overview %}.